File: Ecity/Category/NSUserDefaults+EcityDefaults.m
//
//  NSUserDefaults+EcityDefaults.m
//  Ecity
//
//  Created by wangwenbing on 2017/4/25.
//  Copyright © 2017年 Centit. All rights reserved.
//

#import "NSUserDefaults+EcityDefaults.h"

NSString *kEcityCompletedFirstRun = @"kEcityCompletedFirstRun";

@implementation NSUserDefaults (EcityDefaults)
    
- (void)ecity_regeisterDefaults {
    NSDictionary *defaultsInfo = @{ kEcityCompletedFirstRun : @(YES)
                                    
                                    };
    [self registerDefaults:defaultsInfo];
}
@end

File: Ecity/Category/NSUserDefaults+EcityDefaults.swift
//
//  NSUserDefaults+EcityDefaults.swift
//  Ecity
//
//  Created by wangwenbing on 2017/4/27.
//  Copyright © 2017年 Centit. All rights reserved.
//

import Foundation

let kEcityCompletedFirstRun = "kEcityCompletedFirstRun"

extension UserDefaults
{
    public func ecity_registerUserDefaults() -> Void {
        let defaultsInfo = [kEcityCompletedFirstRun:(true)]
        self.register(defaults: defaultsInfo)
        
    }
}
